src/_sass/components/videoplayer/_score-percussion.scss

```diff
@@ -3,38 +3,75 @@
   background-position: center;
 }
 
-.cell--percussion-blci {
-  background-image: url('./images/note-chin.svg');
-}
+.cell--percussion-bcbc { background-image: url('./images/note-bcbc.svg'); }
+.cell--percussion-bcwc { background-image: url('./images/note-bcwc.svg'); }
+.cell--percussion-bcwt { background-image: url('./images/note-bcwt.svg'); }
+.cell--percussion-btbc { background-image: url('./images/note-btbc.svg'); }
+.cell--percussion-btko { background-image: url('./images/note-btko.svg'); }
+.cell--percussion-btwc { background-image: url('./images/note-btwc.svg'); }
+.cell--percussion-btwt { background-image: url('./images/note-btwt.svg'); }
+.cell--percussion-chun { background-image: url('./images/note-chun.svg'); }
+.cell--percussion-dain { background-image: url('./images/note-dain.svg'); }
+.cell--percussion-don { background-image: url('./images/note-don.svg'); }
+.cell--percussion-hobc { background-image: url('./images/note-hobc.svg'); }
+.cell--percussion-hochu { background-image: url('./images/note-hochu.svg'); }
+.cell--percussion-howc { background-image: url('./images/note-howc.svg'); }
+.cell--percussion-ibc { background-image: url('./images/note-ibc.svg'); }
+.cell--percussion-iyapu { background-image: url('./images/note-iyapu.svg'); }
+.cell--percussion-kata { background-image: url('./images/note-kata.svg'); }
+.cell--percussion-kbci { background-image: url('./images/note-kbci.svg'); }
+.cell--percussion-kesu { background-image: url('./images/note-kesu.svg'); }
+.cell--percussion-kopu { background-image: url('./images/note-kopu.svg'); }
+.cell--percussion-kwci { background-image: url('./images/note-kwci.svg'); }
+.cell--percussion-kwtr { background-image: url('./images/note-kwtr.svg'); }
+.cell--percussion-obc { background-image: url('./images/note-obc.svg'); }
+.cell--percussion-obci { background-image: url('./images/note-obci.svg'); }
+.cell--percussion-osae { background-image: url('./images/note-osae.svg'); }
+.cell--percussion-owtr { background-image: url('./images/note-owtr.svg'); }
+.cell--percussion-shon { background-image: url('./images/note-shon.svg'); }
+.cell--percussion-wtbc { background-image: url('./images/note-wtbc.svg'); }
+.cell--percussion-wtko { background-image: url('./images/note-wtko.svg'); }
+.cell--percussion-wtwc { background-image: url('./images/note-wtwc.svg'); }
+.cell--percussion-wtwt { background-image: url('./images/note-wtwt.svg'); }
+.cell--percussion-yobc { background-image: url('./images/note-yobc.svg'); }
+.cell--percussion-yochu { background-image: url('./images/note-yochu.svg'); }
+.cell--percussion-yoibc { background-image: url('./images/note-yoibc.svg'); }
+.cell--percussion-yopu { background-image: url('./images/note-yopu.svg'); }
+.cell--percussion-yowi { background-image: url('./images/note-yowi.svg'); }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Do we need all these blank lines?

Reply:
Fixed. I didn't realize that those blank lines were there.